cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -91,6 +90,17 @@ def counts_to_results(
         # Appends all the keys onto 'samples' list number-of-counts-in-the-key times
         # If collections.Counter({"01": 48, "11": 52}), [0, 1] is appended to 'samples` 48 times and
         # [1, 1] is appended to 'samples' 52 times
+        if int(counts_of_key) != counts_of_key:
+            print(
+                warnings.warn(
```

Comment:
we probably don't want to warn for every key - maybe we could just precede the loop with a single check like
```python
if not all(counts == int(counts) for counts in counter.values()):
    warnings.warn(...)
```

Reply:
Done.